src/library_html5_webgl.js

```diff
@@ -91,6 +91,16 @@ var LibraryHtml5WebGL = {
       renderViaOffscreenBackBuffer: HEAP8[attributes + {{{ C_STRUCTS.EmscriptenWebGLContextAttributes.renderViaOffscreenBackBuffer }}}]
     };
 
+
+#if ASSERTIONS
+    assert(contextAttributes.majorVersion === 1 || contextAttributes.majorVersion === 2, `${contextAttributes.majorVersion} is not a valid value for contextAttributes.majorVersion (should be 1 or 2)`);
```

Comment:
How about just `invalid GL version requested: ${contextAttributes.majorVersion}`.   

I tend to prefer shorted error message, but I get that not everyone agrees with me :)